editing/cefalse-beginning-deletion-manual.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Selecting and deleting all from the cE=true element with cE=false element at the beginning</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  #host {
+    border: 2px red solid;
+  }
+</style>
+<div contenteditable id="host">
+  <div contenteditable="false" id="cefalse">&nbsp;</div>
+  <p id="paragraph">Lorem ipsum dolor sit amet.</p>
+</div>
+
+<ol>
+  <li>Select all from the red area above.</li>
+  <li>Delete it by pressing <kbd>Backspace</kbd>/<kbd>Delete</kbd>.</li>
+</ol>
+<script>
+  setup( { explicit_timeout: true } );
+
+    promise_test( () => {
+    const cefalse = document.querySelector( '#cefalse' );
+    const paragraph = document.querySelector( '#paragraph' );
+    return new Promise( ( resolve, reject ) => {
+        host.addEventListener( 'keyup', ( { key }) => {
+          if ( key !== 'Backspace' && key !== 'Delete' ) {
+            return;
+          }
+
+          try {
+              assert_false( cefalse.isConnected, 'cE=false element was removed' );
+              assert_false( paragraph.isConnected, 'paragraph was removed' );
+          } catch ( error ) {
+              reject( error );
+          }
+          resolve();
+        } );
+    } );
```

Comment:
It seems that the following code gets same result as I test this manually:
```
    const cefalse = document.querySelector( '#cefalse' );
    const paragraph = document.querySelector( '#paragraph' );
    getSelection().collapse(paragraph, 0);
    document.execCommand("selectAll");
    document.execCommand("delete");
    assert_false( cefalse.isConnected, 'cE=false element was removed' );
    assert_false( paragraph.isConnected, 'paragraph was removed' );
```
And the other manual tests can be rewritten with this style.